.buildkite/scripts/packaging/packaging.sh

```diff
@@ -12,8 +12,12 @@ set -ueo pipefail
 
 BEAT_DIR=${1:?-"Error: Beat directory must be specified."}
 
-#Use newer multiarch support for packaging
-docker run --privileged --rm tonistiigi/binfmt:master --install all
+# NOTE: Pinned to QEMU v8.x since v9.x breaks compilation of amd64 binaries in arm64.
+# See: https://gitlab.com/qemu-project/qemu/-/issues/2560
+BINFMT_IMAGE="tonistiigi/binfmt:qemu-v8.1.5"
+# Make sure to uninstall first to avoid conflicts
+docker run --privileged --rm "$BINFMT_IMAGE" --uninstall qemu-*
```

Comment:
shall this run as a best effort? something like `|| true`

Reply:
@v1v never mind this PR it's for testing some things out -- I will close it soon. Btw the above command is always guaranteed to work.